### DISCUSSION TITLE
Two Factor Authentication -  TOTP

### DISCUSSION THREAD

**Discussion by russ-sentient** [mealie-recipes/mealie#2321] [Feature Request]
It would be a nice option to be able to secure login with a 2 Factor Auth., easiest being TOTP through an authenticator app.

**Comment by michael-genson** [comment]
We currently support LDAP and OIDC authentication:
LDAP: https://docs.mealie.io/documentation/getting-started/authentication/ldap/
OIDC: https://docs.mealie.io/documentation/getting-started/authentication/oidc/

You can use 2FA through OIDC

**Comment by skank01** [comment]
Would love 2 factor authentication right away too..
Using google authenticator.
